flink/flink.sh

```diff
@@ -133,11 +138,34 @@ EOF
 
 }
 
+function install_flink_snapshot() {
+  local FLINK_TOPLEVEL='flink-1.5.0'
```

Comment:
Is the 1.5.2 toplevel dir really 1.5.0?

Reply:
good catch.  I must have missed this during testing.  I will update and do another manual sanity check.